index.js

```diff
@@ -22,11 +22,15 @@ const baseLink = "https://www.icloud.com/shortcuts/";
 class Action {
 	constructor(action) {
 		/**
-		 * @type {string} A namespace of the action.
+		 * A namespace of the action.
+		 * @type {string}
 		 */
 		this.identifier = action.WFWorkflowActionIdentifier;
 
 		/**
+		 * An object denoting a shortcut action.
+		 * The propeties in the object are not renamed and follow the appple naming convention.
+		 * Example: {"WFCommentActionText":"example value"}
```

Comment:
I do not think this example is necessary.

Reply:
Ok